### PR TITLE
CDC-2400 Added readme  and flows for the usage of contact priority

### PR DESCRIPTION
**Pull request by prsetiya**
CDC-2400 CDC-2400 Added readme  and flows for the usage of contact priority

### CODE REVIEW

File: Webex Connect Flows/v3.1/Sample/Usage Of Surveys in Flows/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (keshshar) [inline Webex Connect Flows/v3.1/Sample/Usage Of Surveys in Flows/README.md]: This needs to be v3.2

Comment (prsetiya) [reply Webex Connect Flows/v3.1/Sample/Usage Of Surveys in Flows/README.md]: Sure

Comment (prsetiya) [reply Webex Connect Flows/v3.1/Sample/Usage Of Surveys in Flows/README.md]: Done

File: Webex Connect Flows/README.md [hunk 2]
```diff
@@ -49,7 +49,8 @@ In 3.1 version of workflows, the flows are enhanced with Security scan results t
...
-| Workflows version | High level Description                      |
-| ----------------- | ------------------------------------------- |
-| 3.1               | Simplified flows with Security scan results |
-| 3.0               | Simplified flows                            |
\ No newline at end of file
+| Workflows version | High level Description                                |
+|-------------------|-------------------------------------------------------|
+| 3.2               | Simplified flows with Survey and Set Contact Priority |
+| 3.1               | Simplified flows with Security scan results           |
+| 3.0               | Simplified flows                                      |
```
Comment (siramana) [inline Webex Connect Flows/README.md]: 3.0 -> The 2.x flows undergone a major update in 3.0 where the contact center logic in contact resolve, routed, modified, closed usecases are simplified with new nodes. 

3.1 -> Malware scanning feature is integrated to all the attachments send in all channels. The channel start nodes are enhanced to have new set of variables to identify the malware and security scan results.

3.2 -> Flow nodes are enhanced to configure contact priority and allow system messages like Survey links, Thank you messages gets delivered to Customer.

Can you please check and rephrase accordingly.

File: Webex Connect Flows/v3.2/Sample/Usage Of Surveys in Flows/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+-  Pre-requisites :- 
+1. You must configure the livechat main flow's resolve conversation node with "override default contact close event handling" checkbox being checked.
+
+<br><img width="800" alt="ResolveConvNode" src="../../images/ResolveConvNode.png"><br>
+
+2. In the livechat main flow add the set variable node to set threadId and userId
+
+<br><img width="800" alt="SetVariableLiveChat" src="../../images/SetVariableLiveChat.png"><br>
+
+3. Extract the thread and user in the evaluate node of the LiveChatClose flow.
```
Comment (siramana) [inline Webex Connect Flows/v3.2/Sample/Usage Of Surveys in Flows/README.md]: Can you add the detail why thread and user id are required

**Review by siramana — COMMENTED** [review]
We should have one -> How to manually upgrade from v3.1 to v3.2 workflows

File: Webex Connect Flows/README.md (hunk 2 above)
Comment (prsetiya) [reply Webex Connect Flows/README.md]: Sure

File: Webex Connect Flows/v3.2/Sample/Usage Of Surveys in Flows/README.md (hunk 3 above)
Comment (prsetiya) [reply Webex Connect Flows/v3.2/Sample/Usage Of Surveys in Flows/README.md]: Sure

**Review by keshshar — APPROVED** [review]

**Pull request merged**